.azure-devops/nova-facade-release.yml

```
@@ -14,52 +21,71 @@ variables:
   - name: tags
     value: production,externalfacing
   - name: serviceTreeID
-    value: 6F8CD842-E117-412F-BAE4-56A3B6166594
+    value: ade7d667-42f5-485a-91a9-f1dc6482a9b0
   - name: adoNpmFeedBaseUrl
     value: https://pkgs.dev.azure.com/domoreexp/_apis/packaging/feeds/npm-mirror
 
-jobs:
-  - job: compliance
-    displayName: Compliance checks
-    pool:
-      name: 1ES-Teams-Windows-2022-DomoreexpGithub
-    steps:
-      - template: ./steps/service-tree.yml
-        parameters:
-          serviceTreeID: $(serviceTreeID)
-      - template: ./steps/compliance-steps.yml
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: Azure-Pipelines-1ESPT-ExDShared
+        image: windows-2022
+        os: windows
```

nitpick: you can define pool for our custome jobs and do not need to define it per job
```suggestion
  parameters:
    pool: 
        name: Azure-Pipelines-1ESPT-ExDShared
        image: ubuntu-latest
        os: linux
    sdl:
      sourceAnalysisPool:
        name: Azure-Pipelines-1ESPT-ExDShared
        image: windows-2022
        os: windows
```

---

We only have one job now so I think the outcome it's the same. Previously we've had different pools for different jobs ( compliance - windows / release - linux )